draft-irtf-cfrg-vdaf.md

```diff
@@ -1785,15 +1802,29 @@ def encode_prep_msg(Prio3, k_joint_rand_check):
     return encoded
 
 def decode_prep_msg(Prio3, encoded):
-    k_joint_rand_check = None
-    if Prio3.Flp.JOINT_RAND_LEN > 0:
-        l = Prio3.Prg.SEED_SIZE
-        k_joint_rand_check, encoded = encoded[:l], encoded[l:]
+    if Prio3.Flp.JOINT_RAND_LEN == 0:
+        if len(encoded) != 0:
+            raise ERR_DECODE
+        return None
+    l = Prio3.Prg.SEED_SIZE
+    k_joint_rand_check, encoded = encoded[:l], encoded[l:]
     if len(encoded) != 0:
         raise ERR_DECODE
     return k_joint_rand_check
 ~~~
-{: #prio3-helpers title="Helper functions required for Prio3."}
+
+#### Constants
```

Comment:
Moving these up, next to the variables, would help with understanding the code.

Reply:
Good call. They are referenced in multiple subsections, so I put these at the top of the Prio3 section.